tests/nested/manual/preseed/task.yaml

```diff
@@ -66,6 +66,19 @@ execute: |
   execute_remote "cat /var/lib/snapd/system-key" > system-key.real
   diff -u -w system-key.real system-key.preseeded
 
+  execute_remote "snap debug seeding" | MATCH "preseeded:\s+true"
+  execute_remote "snap debug seeding" | MATCH "seeded:\s+true"
+  # FIXME: this just checks that the time is of the form "xxx.xxxs", which could
+  # break if the preseeding takes more than 60s and golang formats the 
+  # time.Duration as "1m2.03s", etc. but for now this should be good enough
+  execute_remote "snap debug seeding" | MATCH "image-preseeding:\s+[0-9]+\.[0-9]+s"
+  execute_remote "snap debug seeding" | MATCH "seed-completion:\s+[0-9]+\.[0-9]+s"
+
+  # we should not have had any system-key difference as per above, so we 
+  # shouldn't output the preseed system-key or the seed-restart-system-key
+  execute_remote "snap debug seeding" | NOMATCH "preseed-system-key:"
+  execute_remote "snap debug seeding" | NOMATCH "seed-restart-system-key:"
+
```

Comment:
Given that I haven't cherry-picked all the spread tests changes, this commit could be omitted, but I'm not sure I can easily do that without redoing this PR

Reply:
You can commit a revert of this specific commit. Not sure if that will conflict with the world though.

Reply:
Yeah I know of git revert, but not sure about merging into master later.

Reply:
Yeah, this does not work. It looks like we just don't install the right "snapd" inside the nested system?

Reply:
Yes, that was fixed in one of Ian's PRs; but there are older changes that would need to be backported too for these tests to work.

Reply:
Yeah, should be fine, thank you